sailfishusermanager.cpp

```diff
@@ -162,6 +165,9 @@ bool SailfishUserManager::makeHome(const QString &user)
 
 uint SailfishUserManager::addUser(const QString &name)
 {
+    if (!hasAccessRights(SAILFISH_UNDEFINED_UID))
```

Comment:
Either comment here or in the hasAccessRights method that SAILFISH_UNDEFINED_UI is a special case used to just check that the user/process has the required rights.